chainer/backends/_chainerx.py

```diff
@@ -102,6 +102,11 @@ def send_array(self, array):
     def use(self):
         chainerx.set_default_device(self.device)
 
+    def is_array_compatible(self, array):
+        return (
+            isinstance(array, chainerx.ndarray)
+            and self.device == array.device)
```

Comment:
The name is too close to `chainer.is_arrays_compatible` to have a different meaning.

Do you think `numpy.ndarray` is compatible with `chainer.get_device('native:0')`?

Reply:
> Do you think `numpy.ndarray` is compatible with `chainer.get_device('native:0')`?
I don't think so because ChainerX device does not handle numpy array directly. It seems the term "compatible" is too ambiguous. `is_acceptable`?

@niboshi Any idea?

Reply:
Hmm, how about `is_array_supported`?
There's `Device.supported_array_types` and its meaning is the same as this one.

Reply:
I think it is great! I’ll fix it.